src/Standards/Squiz/Tests/WhiteSpace/MemberVarSpacingUnitTest.inc

```diff
@@ -332,3 +332,34 @@ class CommentedOutCodeAtStartOfClassNoBlankLine {
      */
     public $property = true;
 }
+
+class ClassWithAttributes {
+
+    #[Attribute1]
+    public $var1 = 'value';
+
+    #[Attribute1]
+
+
+    #[Attribute2]
+
+    public string $var2 = 'value';
+
+    /**
+     * Description.
+     */
+
+    #[Attribute1]
+    protected $var3 = 'value';
+
+    /**
+     * Description.
+     */
+
+    #[Attribute1]
+
+    #[Attribute2]
+
+
+    private $var3 = 'value';
+}
```

Comment:
Just some suggestions:
* What about a test case where the attribute is _before_ the docblock ?
* What about a test case with one attribute on the same line as the property ?
* What about a test case with two attributes on the same line as the property ?

Would be good to verify that the sniff will handle those cases correctly as well.

Reply:
Despite there is no PSR for Attributes for now, I suggest to always keep phpDoc before attributes.